### DISCUSSION TITLE
560

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#936] [Comments]
# 560

지난 시간에는 EntityQL 환경을 적용해보았습니다. 간단한 예제로 단일 Entity의 Bulk Insert를 보여드렸는데요. 이번 시간에는 OneToMany 환경에서 어떻게 Bulk Insert를 구현할지 알아보겠습니다. 1. 해결책 EntityQL이 전환해주는 Querydsl-SQL은 JPA 기반이 아닙니다. 그래서 OneToMany와 같은 연관관계 Insert/Update 등은 JdbcTemplate처럼 직접 구현을 해야하는데요. 원래 JdbcTemplate으로 작성하던 코드를 단순히 정적 타입 개발이 가능하도록 지원할 뿐인것 이제는 다들 아시죠? 그래서 꼭 Querydsl-SQL이 아니더라도, JdbcTemplate로 구현한다 하여도, OneToMany 를 BulkInsert를 하려면 다음의 과..

https://jojoldu.tistory.com/560

**Comment by kakaruu** [comment]
어흑.. Spring boot 3 환경에서는 EntityQL을 사용할 수가 없네요 ㅠ
Spring boot 3 환경은 EntityQL 마개조밖에 답이 없을란가요..
